frontend/src/types.ts

```diff
@@ -2596,6 +2596,7 @@ export interface Survey {
         selector: string
         seenSurveyWaitPeriodInDays?: number
         urlMatchType?: SurveyUrlMatchType
+        repeatedActivation?: boolean
```

Comment:
Since this only concerns event-based triggering, I would opt for a more specific name for this field, such as `eventBasedRepeatedActivation`

Reply:
sounds good, I moved `repeatedActivation` to within the `events` object so its scope is clear.